### PR TITLE
docs:updated context docs

### PR DESCRIPTION
**Pull request by guptadeepak8**
Fix-#88
/claim #88

### CODE REVIEW

File: docs/guides/context.md [hunk 1]
```diff
@@ -69,15 +69,29 @@ In this scenario, `value.userId` helps retrieve the `userId` information from th
...
+
+Variables enable a simple way to add dynamic values into GraphQL directives, providing flexibility and adaptability within the schema. Integration of variables into directives enables the creation of more flexible and reusable schema configurations.
+
+```graphql showLineNumbers
+type Query {
+  user(id: ID!) @http(url: "/users/{{args.id}}")
+}
+```
+
+Here, `args.id` is a variable representing the value of the id argument provided in the GraphQL query. During query execution, this variable is replaced with the actual id value, enabling the construction of dynamic URLs within the directive.
```
Comment (tusharmath) [inline docs/guides/context.md]: variables should be set using `var` and not `args`. Create an example using `var`.

File: docs/guides/context.md [hunk 2]
```diff
@@ -91,4 +105,27 @@ type Query {
...
+
+Here's a simple example demonstrating the usage of Mustache templates within a directive:
+
+```graphql
+type Query {
+  user(id: ID!) @http(url: "/users/{{args.id}}")
+}
+```
+
+Here, `{{args.id}}` will be replaced with the actual id argument value when making the API request.
```
Comment (tusharmath) [inline docs/guides/context.md]: I think we can move the mustache template section to the top, without a heading. We could be a little brief about its usage. The rest of the document anyways talks about how we can use mustache templates in Context

**Comment by guptadeepak8** [comment]
@tusharmath made the changes

**Comment by guptadeepak8** [comment]
@tusharmath ready for review

**Review by tusharmath — APPROVED** [review]

**Comment by tusharmath** [comment]
Had to re-write the whole thing, unfortunately.

**Pull request merged**